cpp/cmake/EvalGpuArchs.cmake

```diff
@@ -14,12 +14,14 @@
 # limitations under the License.
 #=============================================================================
 
+enable_language(CUDA)
+
 function(evaluate_gpu_archs gpu_archs)
   set(eval_file ${PROJECT_BINARY_DIR}/eval_gpu_archs.cu)
   set(eval_exe ${PROJECT_BINARY_DIR}/eval_gpu_archs)
   set(error_file ${PROJECT_BINARY_DIR}/eval_gpu_archs.stderr.log)
   file(WRITE ${eval_file}
-    "
+[=[
```

Comment:
What is this?

Reply:
They are called [Bracket Arguments](https://cmake.org/cmake/help/latest/manual/cmake-language.7.html#bracket-argument) but you can consider them to be string literals where no CMake variable expansion occurs.